wgpu-types/src/lib.rs

```diff
@@ -601,17 +779,25 @@ impl Default for ColorWrite {
     }
 }
 
+/// Describes the state of the depth/stencil attachment in a render pipeline.
```

Comment:
This doesn't describe the state of the depth attachment itself, it describes the state of the fixed-function hardware that performs depth and stencil operations